### DISCUSSION TITLE
Ability to change clock type between 24 hour and 12 hour clock

### DISCUSSION THREAD

**Discussion by ghost** [f1multiviewer/issue-tracker#339] [Feature requests] [answered]
My small feature request is an option to switch the clock type from 24 hour to 12 hour clock for the times in the Weekend Schedule or any other place in Multiviewer where time is shown.

So for example, instead of it saying 17:00-18:00, it would say 5:00PM-6:00PM.

**Comment by f1multiviewer** [comment, marked as answer]
This will be in the next release!
